src/pages/components/menu/components/menuLinks/MenuLinks.js

```diff
@@ -0,0 +1,71 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Helpers
+import _map from 'lodash/map';
+import _isEmpty from 'lodash/isEmpty';
+import _startCase from 'lodash/startCase';
+
+//constants
+import { INITIAL_STATE } from './constants/menuLinks.general';
+
+//Components
+import LinkItem from '../../../../../commonComponents/linkItem';
+
+//CSS
+import './menuLinks.css';
+
+class MenuLinks extends React.Component {
+  state = INITIAL_STATE;
+
+  componentDidMount() {
+    const {
```

Comment:
you can place this code in a separate function and call that function from here. that will give more clarity about what we are doing on mount of this component